src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -100,17 +107,69 @@ public void Start()
             }
 
             port = TcpUtil.FindFreePort(DEFAULT_DAEMON_PORT);
-            process = new Process
+            process = new System.Diagnostics.Process
             {
                 StartInfo = new ProcessStartInfo
                 {
                     UseShellExecute = false,
                     FileName = ExePath,
                     Arguments = GetCmdArgs(port),
-                    CreateNoWindow = true
+                    CreateNoWindow = true,
+                    WindowStyle = ProcessWindowStyle.Hidden,
+                    RedirectStandardOutput = true
                 }
             };
+            CreatePane(new Guid(), "SonarLint Daemon", true, false);
+            process.OutputDataReceived += (sender, args) => pane.OutputString(args.Data + "\n");
+            pane.OutputString("Running " + ExePath + "\n");
             process.Start();
+            process.BeginOutputReadLine();
+
+
+
+            channel = new Channel($"{DAEMON_HOST}:{port}", ChannelCredentials.Insecure);
+            daemonClient = new StandaloneSonarLint.StandaloneSonarLintClient(channel);
+
+            ListenForLogs();
+        }
+
+        private async System.Threading.Tasks.Task ListenForLogs()
```

Comment:
Seems I can't: 'Task' is an ambiguous reference between 'Microsoft.VisualStudio.Shell.Task' and 'System.Threading.Tasks.Task'